apps/designer/app/canvas/canvas.tsx

```diff
@@ -27,9 +27,11 @@ import {
   rootInstanceContainer,
```

Comment:
(writing here just to make it a thread)

Bug:

- add a "button" to canvas
- set "type" props to "reset"
- select the button on canvas
- copy (Cmd+C)
- click on canvas to make "body" selected
- paste (Cmd+V)
- the "type" props value is not copied

Same bug when selecting in the navigator instead of canvas

Reply:
Yeah, we need to generate new id for each inserted prop.

Reply:
I approved, though if its a new bug please fix it before merging

Reply:
Kinda new. Right now a lot of duplicated prop.id are stored in db. Here the bug just become visible.

Reply:
I'll fix props and styles copy paste in next PR